react/components/atoms/input-animated/input-animated.js

```diff
@@ -0,0 +1,266 @@
+import React, { PureComponent } from "react";
+import { Animated, StyleSheet, View } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+import { Input, Touchable } from "../../atoms";
+
+export class InputAnimated extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            header: PropTypes.string,
+            borderBottomActiveColor: PropTypes.string,
+            placeholder: PropTypes.string,
+            value: PropTypes.string,
+            onValueUpdate: PropTypes.func,
+            onBlur: PropTypes.func,
+            onFocus: PropTypes.func
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            header: undefined,
+            borderBottomActiveColor: "#4f7af8",
+            placeholder: undefined,
+            value: undefined,
+            onValueUpdate: value => {},
+            onFocus: () => {},
+            onBlur: () => {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.headerTextYPositionBlur = 5;
+        this.headerTextYPositionFocus = 30;
+
+        this.textFontSizeBlur = 16;
+        this.textFontSizeFocus = 12;
+
+        this.headerTextColorBlur = 0;
+        this.headerTextColorFocus = 150;

```

Comment:
I believe if you change this to 1, and below in textInput style `_textStyle` the `inputRange` to `[0, 1]`, the behaviour is the same. Its only weird, since 150 does not make sense, and since you are using interpolate down below, using 0 and 1 equal to the 0-100% logic (which 150 does not)